src/main/java/cz/muni/fi/pa165/dao/RoleDao.java

```diff
@@ -0,0 +1,28 @@
+package cz.muni.fi.pa165.dao;
+
+
+package cz.muni.fi.pa165.dao;
```

Comment:
Duplicitní deklarace package, syntakticky trochu problém - zkoušel jsi to před pushem zkompilovat?